im-gespraech-buber-gesellschaft.csl

```diff
@@ -0,0 +1,428 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Im Gespräch - Hefte der Martin Buber-Gesellschaft (German)</title>
+    <title-short>Im Gespräch</title-short>
+    <id>http://www.zotero.org/styles/im-gespraech-buber-gesellschaft</id>
+    <link href="http://www.zotero.org/styles/im-gespraech-buber-gesellschaft" rel="self"/>
+    <author>
```

Comment:
It is sure that the style is not that usual. In parts this is due to the purpose of this style to give precise reference in cases where the exact edition of a text matters.